lib/matplotlib/axes/_axes.py

```diff
@@ -3952,7 +3961,17 @@ def do_patch(xs, ys, **kwargs):
         elif len(widths) != N:
             raise ValueError(datashape_message.format("widths"))
 
-        for pos, width, stats in zip(positions, widths, bxpstats):
+            # capwidth
```

Comment:
wrong indent?  Probably can just remove this as its pretty obvious next block is capwidths.

Reply:
Fixed the indention, my bad.